uswds/uswds-site
# USWDS-Site: Settings documentation

## Summary
Removed redundant instructions for setting up the Sass file structure from the settings page.

## Related issue
Closes https://github.com/uswds/uswds-site/issues/1701

Recommend releasing this update with the Phase 2 compile update in PR https://github.com/uswds/uswds-site/pull/1766

## Preview link
Preview link: [Settings page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.app.cloud.gov/preview/uswds/uswds-site/al-settings/documentation/settings/)

## Problem statement
Providing the same instructions in two different locations can cause confusion for the user and requires double maintenance. Additionally, users looking to configure their settings will receive more information than needed.

## Solution
Replacing the instructions for setting up Sass files with a link to the Getting Started guide allows the user to focus on only configuring the settings. 

## Testing and review
To review:
- Proofread for errors
- Confirm that instructions makes sense 
- Confirm that instructions are comprehensive

Before opening this PR, make sure you’ve done whichever of these applies to you:
